src/styles/header.scss

```diff
@@ -0,0 +1,68 @@
+@import 'fonts';
+@import 'colors';
+
+.containerHeader {
+  display: flex;
+  width: auto;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the width should not be set to 'auto' as it is the default value. Consider removing this line if there's no need to override a previously set width.